src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+:root {
+  --active-color: #00acdc;
+
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", "Helvetica Neue", Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  width: 1100px;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  text-align: center;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  position: relative;
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.logo {
```

Comment:
Please move this selector to the top so that the css style order is the same as the html code order. It structures your code better, and is more readable

Reply:
Done! Thank you for your feedback